<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table_control</class>
 <widget class="QWidget" name="table_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPushButton" name="Enable_table">
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Joystick enable</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>571</width>
       <height>142</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QPushButton" name="got_to_previous">
        <property name="text">
         <string>Move to previous</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="Y_label">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="y_move">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QDial" name="Table_speed">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="X_label">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="x_move">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="unlock_Z">
        <property name="text">
         <string>Unlock Z-move</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="check_position">
        <property name="text">
         <string>Check position</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLCDNumber" name="Table_speed_indicator">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSlider" name="z_move">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="Z_label">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Rpoly_val">
        <property name="text">
         <string>    Joystick speed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="table_ind">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;
border-radius: 5px;
background-color: grey;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string> Table moving </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Enable_table</sender>
   <signal>clicked(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unlock_Z</sender>
   <signal>clicked(bool)</signal>
   <receiver>z_move</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Table_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Table_speed_indicator</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
